packages/tailwindcss/src/utilities.test.ts

```diff
@@ -12585,10 +12589,6 @@ test('font', async () => {
       font-family: ui-sans-serif;
     }
 
-    .font-sans {
-      font-family: var(--font-sans);
-    }
```

Comment:
Also wouldn't expect this moving.

Reply:
Aha! I accidentally counted properties that were set to `undefined | null`. So we counted `3` for this guy (`font-family`, `font-feature-settings`, `font-variation-settings`)